### ISSUE TITLE
Show Overview page in background after the first time

### ISSUE THREAD

**Issue by akaroml** [Microsoft/vscode-java-pack#51]
Showing overview page every time a Java project is opened could block users' workflow. Here to track the effort of only showing the page in the foreground the first time. And later, the page should be shown in the background when the page is not requested by users.

**Comment by akaroml** [comment]
This depends on Microsoft/vscode#62375
We'll need vscode to provide API to achieve that.

**Comment by kdvolder** [comment]
I would also add that the overview page gets opened *again* even if it is already open, so after starting / restarting vscode a couple of times, unless you close the page each time, you will end up with a ton of 'Java Overview' pages crowding out the other editors in the Editor tabs area. It is mildly annoying to keep closing these pages over and over.

If the page is open, you should probably detecting this and avoid opening a second another copy of the page.

**Comment by akaroml** [comment]
> I would also add that the overview page gets opened _again_ even if it is already open, so after starting / restarting vscode a couple of times, unless you close the page each time, you will end up with a ton of 'Java Overview' pages crowding out the other editors in the Editor tabs area. It is mildly annoying to keep closing these pages over and over.
> 
> If the page is open, you should probably detect this and avoid opening another copy of the page.

I totally agree. This is currently a limitation of vscode webview API. An extension is not able to know whether a webview is already persisted by vscode microsoft/vscode#73017. And now custom editor view is suggested to replace the webview microsoft/vscode#77131, and we'll see how we can adopt this.
